stable/coredns/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.serviceAccount.create }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ template "coredns.fullname" . }}
```

Comment:
See https://github.com/helm/helm/blob/master/docs/chart_best_practices/rbac.md - there is a dedicated template for the serviceaccount - to be used here and in the deployment manifest